spec/services/ccms/requestors/case_add_requestor_xml_blocks/non_passported_additional_property_attributes_spec.rb

```diff
@@ -119,8 +119,8 @@ def hard_coded_attrs
             ["other_savings", "OTHERSAVING_INPUT_T_10WP2_3A", "text", true, "."],
             ["other_savings", "OTHERSAVING_INPUT_T_10WP2_5A", "text", true, "Client Sole"],
             ["cli_premium", "CLIPREMIUM_INPUT_T_9WP2_12A", "text", true, "."],
-            ["proceeding", "PROC_UPPER_TRIBUNAL", "text", false, "No"],
-            ["proceeding", "PROC_UPPER_TRIBUNAL", "text", false, "No"],
+            ["proceeding_means", "PROC_UPPER_TRIBUNAL", "text", false, "No"],
```

Comment:
Was this something you spotted?

Reply:
I amended the XmlExtractor to make it clearer (IMO) what it is checking. `proceeding` entity is actually checking the proceeding in the MeansAssesment Entity, as opposed to the proceeding in the MeritsAssesment entity (typo in assessment deliberate.